elasticdl_preprocessing/layers/to_sparse.py

```diff
@@ -0,0 +1,59 @@
+import tensorflow as tf
+
+
+class ToSparse(tf.keras.layers.Layer):
+    """Converts a `Tensor` to a `SparseTensor`, dropping ignore_value cells.
+    If the input is already a `SparseTensor`, just return it.
+
+    Example :
+    ```python
+        layer = ToSparse()
+        inp = tf.constant([["A", ""], ["B", "C"]], tf.string)
+        layer.call(inp)
+        tf.SparseTensor(
```

Comment:
why is this in the example?

Reply:
Should probably mention this is the expected output

Reply:
Yeah